shared/p2p/service_test.go

```diff
@@ -17,13 +17,17 @@ import (
 	swarmt "github.com/libp2p/go-libp2p-swarm/testing"
 	shardpb "github.com/prysmaticlabs/prysm/proto/sharding/p2p/v1"
 	testpb "github.com/prysmaticlabs/prysm/proto/testing"
-	"github.com/prysmaticlabs/prysm/shared"
 	"github.com/sirupsen/logrus"
 	logTest "github.com/sirupsen/logrus/hooks/test"
 )
 
+type testService interface {
```

Comment:
Why do you need this? This is the same interface as shared.Service

Reply:
I cant use that due to cyclical imports, the go compiler would not be able to compile then . So I just created a similar one here